src/entries/popup/pages/home/Activity/ActivitiesList.tsx

```diff
@@ -44,6 +45,7 @@ export function Activities() {
   } = useInfiniteTransactionList({
     getScrollElement: () => containerRef.current,
   });
+  useTransactionListForPendingTxs();
```

Comment:
since using hooks for "watching" purposes it's not very common pattern, I'd add a comment saying that here.